### ISSUE TITLE
the tool still download data from onedrive to local before saving them into shared drive.

### ISSUE THREAD

**Issue by MrC1rcle** [Image-X-Institute/The-Real-Time-Imaging-Database#24]
The size of data the needs to transmit into shared drive are really large, some of the folders may be over 100GB. the local drive may not large enough to save so much data during the transmission.

**Comment by ChandrimaSengupta** [comment]
I think this is a Onedrive issue. As discussed previously with Indrajit, this is not solvable? @Gahhh?

**Comment by Gahhh** [comment]
> I think this is a Onedrive issue. As discussed previously with Indrajit, this is not solvable? @Gahhh?

Yes this issue was from OneDrive, but we may have an alternative solution, eg: auto-download content from onedrive to a temp folder in the server, then we start from that.
